### ISSUE TITLE
Stop AppPool - Environment Block [...] cannot be longer than 65535bytes

### ISSUE THREAD

**Issue by mluening** [microsoft/azure-pipelines-extensions#813]
hello guys,

i often get the problem that the task "WinRM - IIS Web App Management" which is normally used to stop my app pools runs into an exception as you can see in the log file:
"[error]Unable to find type [VstsTaskSdk.TerminationException]"

it seems that the environment variables are too long because of 35 build artifacts.
is there any solution for this problem?

[tasklog_38.log](https://github.com/microsoft/azure-pipelines-extensions/files/4586935/tasklog_38.log)

**Comment by asranja** [comment]
Yes you are correct, having a large number of build artifacts adds a lot of environment variables for each artifact. The error is thrown by PowerShell process while trying to execute the task when the env variables size cross the limit.

Currently we do not have a proper solution for this other than reducing the number of artifacts.
